CHANGES

```diff
@@ -238,7 +238,7 @@ Version 4.0.0 (27 Apr 2019)
 	    [PHP] Fix access to already released memory during PHP module
 	    shutdown, which often didn't cause visible problems, but could
 	    result in segmentation faults, bus errors, etc.  Fixes #1170,
-	    reported by Jitka Plesn�kov�.
+	    reported by Jitka Plesn�kov�.
```

Comment:
It looks like this patch changes lines with accented characters - are these changes intended?  The github patch view doesn't seem to know the right encoding so I just see `?` replacement characters before and after...